### PR TITLE
change weird example description

### PR DESCRIPTION
**Pull request by psychocoderHPC**
change weird example description

thx @Flamefire:
https://github.com/ComputationalRadiationPhysics/contributing/issues/2#issuecomment-159548776

### CODE REVIEW

**Comment by Flamefire** [comment]
Typo in "weird". I guess no wires are attached ;)

**Comment by psychocoderHPC** [comment]
@Flamefire  thx I fixed it

File: codingGuideLines/cpp.md [hunk 1]
```diff
@@ -465,7 +465,7 @@ template<
 >
 class Bob;
 ```
-  - for C++11 `using` new line after assign `=` mandatory ( definition is indented )
+  - partly specialization using the C++11 keyword `using`
```
Comment (ax3l) [inline codingGuideLines/cpp.md]: typo: *partial* specialization

**Comment by ax3l** [comment]
@psychocoderHPC let us rebase & merge this?

**Comment by psychocoderHPC** [comment]
updated, sry for the small time delay of 0.75 years

**Pull request merged**